### ISSUE TITLE
broken URI fragments on http://twistedmatrix.com/documents/current/

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#5736]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)| @thijstriemstra reported|
|-|-|
|Trac ID|trac#5736|
|Type|defect|
|Created|2012-07-02 10:52:51Z|

The link checker [points out](http://validator.w3.org/checklink?uri=http%3A%2F%2Ftwistedmatrix.com%2Fdocuments%2Fcurrent&hide_type=all&depth=&check=Check) 2 broken URI fragments at http://twistedmatrix.com/documents/current/.

Attachments:

* [broken-uri-5736.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/3c6/3c69c4b61641208b08dc9ca9b70bdfb4d483fb64/94cb5a0e1d2bf4ecea4b0e43594fc0cd89b8959a.patch) (631 bytes) - added by thijs on 2012-07-02 10:54:35Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__5736 5736
type__defect defect
reporter__thijs thijs
priority__low low
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation_easy documentation easy
time__1341226371000000 1341226371000000
changetime__1342354847000000 1342354847000000
version__None None
owner__thijs thijs
cc__thijs cc__jesstess
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)|@thijstriemstra removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_2"></a>|@exarkun commented|
|-|-|

Don't use commas in the trac keyword field.  They don't work.

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)<a name="note_4"></a>|@jesstess set owner to @thijstriemstra|
|-|-|

Thanks for catching and fixing these. Looks good, please merge!

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_5"></a>|@thijstriemstra set status to `closed`|
|-|-|

(In [34836]) Apply broken-uri-5736.patch: Correct broken links for the historical documents.

Author: thijs
Reviewer: jesstess
Fixes: [#5736](https://github.com/twisted/twisted/issues/5736)
